code/__DEFINES/reagents.dm

```diff
@@ -2,7 +2,9 @@
 #define LIQUID 2
 #define GAS 3
 
-#define OPENCONTAINER	4096	// is an open container for chemistry purposes
+#define INJECTABLE		1024	//Makes reagents addable through droppers and syringes
+#define DRAWABLE		2048	//If a syringe can draw from it
```

Comment:
Aren't containers using any of these flag slots or was it just a lazy change leaving OPENCONTAINER at 4096?